scalableminds/webknossos
# Multiple custom layouts for tracing views

This PR addes the possibility to add muliple layouts.

### URL of deployed dev instance (used for testing):
- https://mulitpletracingcustomlayouts.webknossos.xyz/

### Steps to test:
- open a tracing
- create a new layout (should have the previous layout set as initial layout)
- change freely between the layouts
- try deleting a non-active layout
- an active layout should not be removable -> thus guaranteeing to always have a layout 
- the last active layout at closing a tracing, should be the initial layout on next startup
- important -> try this in all four kinds of tracing views as their layout is treated separately
- test the migration into the new "layout saving schema"

### Issues:
- fixes #3230

------
- ~~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~~
- ~~~[ ] Updated [documentation](../blob/master/docs) if applicable~~~
- ~~~[ ] Needs datastore update after deployment~~~
